config/default.yml

```diff
@@ -114,6 +114,12 @@ CommitMsg:
     enabled: true
     description: 'Check for trailing periods in subject'
 
+  Commitplease:
+    enabled: false
+    description: 'Analyze with Commitplease'
+    required_executable: './node_modules/.bin/commitplease'
```

Comment:
I've quickly skimmed the thread https://github.com/jzaefferer/commitplease/issues/72#issuecomment-235827024 and I'm not entirely clear on why you are accessing this via a local `node_modules` directory instead of via your `PATH`. If you look at the `EsLint` hook, we refer to the [`eslint` executable](https://github.com/brigade/overcommit/blob/8edb55305b236908cf74162f0a05dba976ab0609/config/default.yml#L234-L236) instead of its path within the `node_modules` directory.

Has this been tested? Does it work?